recipes/ctds/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ctds" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "f94e36e277ba81aeb02c2266289eb4d576e727d37810fa534eb3b6f424b18a96" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and py >= 35]
+  skip: True  # [win]
```

Comment:
You can probably skip the two build files and do a multiline `script` like this:

```yaml
script:
  - python setup.py build_ext --include-dirs=$PREFIX/include --library-dirs=$PREFIX/lib  # [not win]
  - python setup.py install --single-version-externally-managed --record record.txt